packages/block-library/src/post-template/style.scss

```diff
@@ -19,24 +19,18 @@
 		flex-direction: row;
 		display: flex;
 		flex-wrap: wrap;
-
+		gap: 1.25em;
 		li {
-			margin: 0 0 1.25em 0;
-			width: 100%;
+			flex: 1;
+			min-width: 250px;
+			margin: 0;
 		}
 
-		@include break-small {
-			li {
-				margin-right: 1.25em;
-			}
-
+		@include break-xlarge {
```

Comment:
This change effectively switches off the column count for all viewport widths less than 1080px, so if someone has set their Query Loop block to a column count of `2` for example, then in narrower viewports, 3 or 4 columns will be displayed instead.

Reply:
you are right here, i haven't notice this. this can be fixed. thanks.